frontend/src/components/playlists/PlaylistItem.tsx

```diff
@@ -0,0 +1,35 @@
+import React, { FC } from 'react';
```

Comment:
Is `FC` typescript for react ?

Reply:
Yes it stands for `Functional Component`, it's a generic interface accepting the Props of your component as `FC<Props>` to validate the input types